ci-operator/step-registry/openshift/e2e/fips/openshift-e2e-fips-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: openshift-e2e-fips
+  from: cli
+  commands: openshift-e2e-fips-commands.sh
+  env:
+  - name: TMPDIR
+    default: /tmp
+    documentation: A pathname of a directory made available for programs that need a place to create temporary files.
+  - name: FIPS_EXPECTED
```

Comment:
hmm.. this env is not used ..

Reply:
Oops, I need to set it in the workflow that enables FIPS...